src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -35,13 +38,20 @@ export class MultisigTransactionInfoMapper {
 
   constructor(
     @Inject(ITokenRepository) private readonly tokenRepository: TokenRepository,
+    @Inject(IConfigurationService)
+    private readonly configurationService: IConfigurationService,
     private readonly dataDecodedParamHelper: DataDecodedParamHelper,
     private readonly customTransactionMapper: CustomTransactionMapper,
     private readonly settingsChangeMapper: SettingsChangeMapper,
     private readonly nativeCoinTransferMapper: NativeCoinTransferMapper,
     private readonly erc20TransferMapper: Erc20TransferMapper,
     private readonly erc721TransferMapper: Erc721TransferMapper,
-  ) {}
+    private readonly humanDescriptionMapper: HumanDescriptionMapper,
+  ) {
+    this.HUMAN_DESCRIPTION_ENABLED = this.configurationService.get(
+      'features.humanDescription',
+    );
```

Comment:
Since this configuration should always exist we can use `getOrThrow`.

Even if the environment variable is not set it would have a value of `false`.

Reply:
This would avoid having an `undefined` value and additional branches to deal with `undefined`.